### ISSUE TITLE
Create RFC-0010 Node software updates

### ISSUE THREAD

**Issue by fflush** [internet-sicherheit/eco-blockchain-governance#31]
Creation of the RFC according to the Title.

**Comment by hphu22483** [comment]
Software update

**Comment by hphu22483** [comment]
> Creation of the RFC according to the Title.

Android phone

**Comment by kiview** [comment]
haha, we assume this was a bot

**Comment by cre8** [comment]
@kiview so each kown member of this repo is a collaborator, isn't he? Maybe we can limit the write access to issues to them to prevent some kind of spamming

**Comment by kiview** [comment]
This has been changed to a public repo last week.
Everyone should be to comment on issues and PRs. We will deal with spammers, once this becomes a problem.

**Comment by fflush** [comment]
This is a good addition. However, only short, concise sentences or key points should be described in the specifications. But it is very difficult to find good examples and good practices there. I would rather see the points currently in the specifications as best practices, as they have already been formulated very concretely. 

I would extract key points for the specifications. So move the current content. Would that be okay? Otherwise please just make other suggestions.

**Comment by kiview** [comment]
Was this an answer to the spam bot? 😄

**Comment by fflush** [comment]
No, that was meant for Mirko. This is about the last commit. :)

**Comment by kiview** [comment]
Ah, I see. You can also comment directly on commits:
https://github.com/internet-sicherheit/eco-blockchain-governance/commit/582ba176aabdf81f6d7aef7a58cf4200203d6bb9

**Comment by fflush** [comment]
Okay, well, I'll see. Then I'll put more general stuff here and specific stuff directly in the commit. Thanks

**Comment by NiehuesP** [comment]
Some thoughts from me about node software updates:

Secure and reliable source for the software?
Internal autority has to test and release the software
Make sure that older version are still available, independently from internet/community software repository
Common test environment must be set and available.
Make sure that not all nodes update at the same time.
Make sure that all nodes finish their update at a final date.
Inform customers about updates
